src/main/java/com/apollocurrency/aplwallet/apl/http/EventListener.java

```diff
@@ -453,15 +454,15 @@ public void onTimeout(AsyncEvent event) {
             pendingWaits.remove(context);
             JSONObject response = new JSONObject();
             response.put("events", new JSONArray());
-            response.put("requestProcessingTime", System.currentTimeMillis()-timestamp);
+            response.put("requestProcessingTime", NtpTime.getTime()-timestamp);
             try (Writer writer = context.getResponse().getWriter()) {
                 response.writeJSONString(writer);
             } catch (IOException exc) {
                 LOG.debug(String.format("Unable to return API response to %s: %s",
                                                      address, exc.toString()));
             }
             context.complete();
-            timestamp = System.currentTimeMillis();
+            timestamp = NtpTime.getTime();
```

Comment:
Use ntp time from APL and remove NtpTime singleton. Make getTime method non-static.